cli/transaction/forms.py

```diff
@@ -0,0 +1,46 @@
+"""
+Provide forms for command line interface's transaction commands.
+"""
+from marshmallow import (
+    Schema,
+    fields,
+    validate,
+)
+
+from cli.generic.forms.fields import (
+    FamilyNameField,
+    NodeURLField,
+    ReverseField,
+    TransactionIdentifierField,
+    TransactionIdentifierListField,
+)
+
+
+class GetListTransactionForm(Schema):
```

Comment:
`GetListTransactionForm` > `GetListOfTransactionsForm` or `GetTransactionsListForm` (it's better).

Reply:
@dmytrostriletskyi, done.